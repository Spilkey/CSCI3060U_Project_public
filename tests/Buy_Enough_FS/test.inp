login
user02
buy
airpods meetup
user01
4
yes
logout
exit