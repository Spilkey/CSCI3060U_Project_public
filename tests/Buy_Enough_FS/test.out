04 airpods meetup            user01          022 100.00
00